README.md

```diff
@@ -160,7 +160,7 @@ For Windows (`*.nupkg`), install with `choco.exe`:
 
 ```shell
 # Install
-choco.exe install zsv --pre -source .\zsv-amd64-windows-mingw.nupkg
+choco.exe install zsv --pre -source .
```

Comment:
Thoughts on doing something like `<directory containing .nupkg file>`? Otherwise I'm not sure if `.` implies that much

Reply:
Yes, I think changing it to that makes sense.